### DISCUSSION TITLE
Vitest and @vue/test-utils error with useRuntimeConfig

### DISCUSSION THREAD

**Discussion by alex129** [nuxt/nuxt#16718] [Questions]
Hello I'm trying to make a simple test with my index.vue page

But I'm having error:

<img width="545" alt="image" src="https://user-images.githubusercontent.com/10452211/193057171-29323021-ac7c-49fc-9ab3-2be853c7187e.png">

My test:

<img width="437" alt="image" src="https://user-images.githubusercontent.com/10452211/193057431-ad6056ca-ed48-46be-9fc1-c1245d522dfd.png">

My index.vue component:

<img width="630" alt="image" src="https://user-images.githubusercontent.com/10452211/193057928-2233ec8f-cf4d-4fca-88a5-2d713ac34829.png">

My vitest.config.js

<img width="368" alt="image" src="https://user-images.githubusercontent.com/10452211/193057615-c2337e4d-f075-45a1-b8fb-14e7dbfceded.png">

I don't know how can I use auto-import or mock env vars

**Comment by sevillaarvin** [comment]
If you are going to test `~/pages` I don't think `@vue/test-utils` is the proper tool. You will have to use `@nuxt/test-utils-edge`

https://v3.nuxtjs.org/getting-started/testing#testing

**Reply by alex129** [reply]
Hello @sevillaarvin but there aren't almost nothing of docu about that, do you have docu about hoy to use it? 

Thanks

**Reply by sevillaarvin** [reply]
Probably because it's still in development:

> Test utils are still in development and the API and behavior may change. Currently, it is in preview stage but not yet ready for testing production apps.
